packages/flutter/lib/src/material/material_app.dart

```diff
@@ -73,6 +73,8 @@ class _MaterialAppState extends State<MaterialApp> implements BindingObserver {
   GlobalObjectKey _navigator;
 
   Size _size;
+  double _devicePixelRatio;
```

Comment:
I just mimicked what we were already doing for size; I don't have a strong opinion.